packages/fuels-core/src/code_gen/mod.rs

```diff
@@ -4,4 +4,6 @@ pub mod abigen;
 pub mod bindings;
```

Comment:
Same comment about `mod.rs`: 

> I believe we've migrated from the mod.rs filename pattern. Not a recommended pattern anymore; see here: https://doc.rust-lang.org/reference/items/modules.html#module-source-filenames.

I know this was already here, but since we're making massive refactors here, might as well.